source/common/io/io_uring_worker.h

```diff
@@ -0,0 +1,509 @@
+#pragma once
+
+#include "source/common/buffer/buffer_impl.h"
+#include "source/common/common/logger.h"
+#include "source/common/io/io_uring.h"
+#include "source/common/io/io_uring_impl.h"
+
+#include "io_uring.h"
+
+namespace Envoy {
+namespace Io {
+
+class IoUringAcceptSocket : public IoUringSocket, protected Logger::Loggable<Logger::Id::io> {
```

Comment:
could also make this inherit from another entity if some of the methods e.g. writev, etc. don't make sense for this entity.

Reply:
Emm...sorry, I'm not sure I get what you mean. Do you mean using another base class here? And that base class hasn't the methods like `writev`?